pyvista/trame/__init__.py

```diff
@@ -0,0 +1,83 @@
+"""Trame interface for PyVista.
+
+This requires trame v2.3.0
+
+"""
+from pyvista import _vtk
+
+import weakref
+from trame.widgets.vtk import VtkLocalView, VtkRemoteView, VtkRemoteLocalView
+
+CLOSED_PLOTTER_ERROR = "The render window for this plotter has been destroyed. Do not call `show()` for the plotter before passing to trame."
+
+
+class _BasePyVistaView:
```

Comment:
I don't think we should add so much logic to an `__init__.py`. I suspect this follows `jupyter/__init__.py`. I want to strip these down and move the main logic into dedicated modules, and only keep the bare minimum (mostly imports) in `__init__.py`.

Reply:
Will do -- I will adjust this trame module here since you're cleaning up the imports in another PR. Should I use `__all__`?

Reply:
That would be nice, yes, thanks. And it might be worth using `_trame.py` to avoid name clashes, pending discussion on the star import PR.

Reply:
Take a look at what I have now. `pyvista.trame` needs to be a publically accessible module so I really don't want to use `pyvista._trame`. Unless you were thinking there'd be an `trame.py` file in that module? In which case, yeah that should be avoided

Reply:
Yeah, sorry, I confused myself a bit earlier. What you have now is right, thanks.